docs/conf.py

```diff
@@ -111,6 +111,11 @@ def _get_project_meta() -> dict[str, str]:
     # OpenAPI types used in TYPE_CHECKING blocks:
     (PY_CLASS, 'SecurityRequirement'),
     (PY_CLASS, 'ExternalDocumentation'),
+    (PY_CLASS, 'Callback'),
```

Comment:
Not sure why this is required?
It should be clear from the annotation, these objects do exist in the docs.